From cd5ecb65a18819eb98e7446e2be35e56dd033534 Mon Sep 17 00:00:00 2001
From: Alex Neundorf <neundorf@kde.org>
Date: Sat, 14 May 2011 22:01:38 +0200
Subject: [PATCH 3/5] -no need to use "REQUIRED" for pkg-config, since the
 result is checked later on anyway via
 find_package_handle_standard_args()

Alex
---
 cmake/modules/FindPolkit.cmake |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/modules/FindPolkit.cmake b/cmake/modules/FindPolkit.cmake
index 5eed321..35fc86e 100644
--- a/cmake/modules/FindPolkit.cmake
+++ b/cmake/modules/FindPolkit.cmake
@@ -21,8 +21,8 @@ if (NOT WIN32)
    # use pkg-config to get the directories and then use these values
    # in the FIND_PATH() and FIND_LIBRARY() calls
    find_package(PkgConfig)
-   pkg_check_modules(PC_POLKIT REQUIRED polkit-gobject-1)
-   pkg_check_modules(PC_POLKIT_AGENT REQUIRED polkit-agent-1)
+   pkg_check_modules(PC_POLKIT polkit-gobject-1)
+   pkg_check_modules(PC_POLKIT_AGENT polkit-agent-1)
    set(POLKIT_DEFINITIONS ${PC_POLKIT_CFLAGS_OTHER})
 endif (NOT WIN32)
 
-- 
1.7.7.4

